includes/class-sensei-usage-tracking.php

```diff
@@ -221,10 +221,10 @@ function maybe_display_tracking_opt_in() {
 				</p>
 				<p>
 					<button class="button button-primary" data-enable-tracking="yes">
-						<?php _e( 'Enable Usage Tracking', 'woothemes-sensei' ) ?>
+						<?php _e( 'Enable Usage Tracking', 'usage-tracking' ) ?>
```

Comment:
The text domain for a shared codebase is the only bit I'm unsure of. @jom @pgk Are there any gotchas here, or is it fine?

Reply:
@alexsanford I wonder if you could ask about this in the #i18n Slack channel just to make sure we're good here?

Reply:
from https://codex.wordpress.org/I18n_for_WordPress_Developers

> In general, an application may use more than one large logical translatable module and a different MO file accordingly. A text domain is a handle of each module, which has a different MO file.

maybe use a less general textdomain name than `usage-tracking`?

Reply:
Based on https://codex.wordpress.org/I18n_for_WordPress_Developers#Text_Domains I’m thinking that adding a new text domain is a bit more work than I want to deal with this close to release :slightly_smiling_face: 

I would like to move the Usage Tracking translations to its own module when I make the whole thing reusable for WPJM. If we release now using the `woothemes-sensei` domain, and later move some strings into the `usage-tracking` (or something like that) domain, then it just means some string translations will have to be moved to the new domain at that point, for each translation. Probably not a huge deal, as it will only be a few strings.

Reply:
I was wondering about the unique text domain here, too. @alexsanford that sounds like a good plan (with possibly a more unique text domain). It might be worth checking how GlotPress handles _shared_ text domains. Since Sensei (Transifex) and WPJM (.org's GlotPress) have different places for managing translations, I'm not sure how helpful it will be either way.